.github/workflows/codeql-analysis.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'go','javascript' ]
+        language: [ 'go' ]
 
```

Comment:
thoughts on moving this one to daily run and have it create issues when it fails?

Reply:
Sgtm, changed it to run daily.

Reply:
do you know how if issues will be created? I think it's fine to do that in another PR entirely, since it's likely to require a bit more thoughts. Sorry about that.

@naveensrinivasan let us know if you'd be OK exploring codeQL daily only.

Reply:
I believe they should be available through the Security tab in Repo settings: https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/managing-code-scanning-alerts-for-your-repository#viewing-the-alerts-for-a-repository

Reply:
You're right. I'm just worried we never check them. I certainly never do :)

Reply:
Why was the `javascript` removed? https://github.com/ossf/scorecard/issues/987

Reply:
👍

Reply:
Wasn't aware of that CodeQL could transitively scan Github actions, I can add Javascript back.

Though, isn't that something scorecard-action can also do via `actionlint`?

Reply:
we'd love to have Dangerous-Workflow check look into Actions (there's an issue for this), but not sure about the dependency Action of the project under scan.

Reply:
Added back `javascript` to CodeQL